waspc/README.md

```diff
@@ -237,16 +237,42 @@ You can do `stack test --coverage` to see the coverage.
 We don't yet have any integration (e2e) tests, but we plan to add them at some point.
 For now, best way is to manually run a Wasp app with `wasp start` and try stuff out.
 
+## Code analysis
 
-## Linting
-While no linter is currently enforced, we recommend using Hlint as a linter.
+To run the code analysis, run:
+```
+./run code-check
+```
+
+This will check if code is correctly formatted, if it satisfies linter, and if it passes static analysis.
 
-If using Hlint as a linter, be aware that Hlint doesn't know which default extensions are we using via Stack/cabal, so it might be missing some extension and therefore report false errors.
+These same checks are required to pass the CI, so make sure this is passing before making a PR.
```

Comment:
This is not yet true but as soon as we get hlint and stan passing we will enable this in CI and then it will be true.